### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_DefaultScreen_Non-Text Contrast: Color contrast ratio of "Ratings" control is less than 3:1.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2941]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;SH_AzureToolsforVSCode_Win32_Aug2021;# Azure Tools for VS
Code;#A11yWCAG2.1;#A11ySev2;#WCAG1.4.11;#Benchmark;#DesktopApp;#Win10;#ColorContrast;

### Environment Details:
Visual Studio Code 
Azure Function
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:
1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open.
4. TAB to "Ratings" controls.
5. Open the AI4D and check the color contrast ratio of the star which shows unchecked state that is filled with White color.

### Actual:
The color contrast ratio of "Rating" control that is star which shows unchecked state is 2.078:1 which is less than 3:1.
![image](https://user-images.githubusercontent.com/54656883/131613835-513a3e22-51c0-4999-af1b-83f85eb817cc.png)

### Expected:
The color contrast ratio of "Rating" control that is star should be greater than or equal to 3:1.

### User Impact:
The low vision users won't be able to see the Rating controls properly.

**Comment by ejizba** [comment]
An extension cannot control the marketplace page. This issue should be filed on VS Code: https://github.com/microsoft/vscode/issues

**Comment by 12-shweta** [comment]
@ejizba  Logged an external bug on provided URL https://github.com/microsoft/vscode/issues/132745

**Comment by InduPriya1805** [comment]
GitHubTags:#BM_AzureToolsforVSCode_Win32_Aug2021;
